ikasaneip/visualisation-dashboard/src/main/java/org/ikasan/dashboard/ui/visualisation/dao/ModuleMetaDataDaoImpl.java

```diff
@@ -51,6 +55,16 @@ private void init()
         return new ArrayList<>(this.moduleMetadata.keySet());
     }
 
+    @Override
+    public List<ModuleMetaData> getAllModule()
+    {
+        JsonModuleMetaDataProvider provider = new JsonModuleMetaDataProvider(new JsonFlowMetaDataProvider());
```

Comment:
Shouldnt provider by initialised as part of the constructor?

Reply:
Can do.

Reply:
done